master/buildbot/www/oauth2.py

```diff
@@ -248,7 +248,14 @@ def __init__(self,
                     'Retrieving team membership information using GitHubAuth is only '
                     'possible using GitHub api v4.')
         else:
-            self.apiResourceEndpoint = self.serverURL + '/graphql'
+            if serverURL is not None:
+                # setup for enterprise github
+                if serverURL.endswith("/"):
```

Comment:
```suggestion
               serverURL = serverURL.lstrip("/"):
```

Reply:
Please put all the onprem vs saas logic near line 231 including computing of the self.graphqlEndpoint